manifest.yml

```diff
@@ -7,4 +7,6 @@ applications:
   services:
   - redis
   path: build/libs/IoTConnectedCarEmulator.jar
-  buildpack: java_buildpack
\ No newline at end of file
+  #buildpack: java_buildpack
+  buildpack: java_buildpack_offline
```

Comment:
I would actually not specify a buildpack...buildpack names between PWS and PCF installs are different...I only added it to make deploys faster for SpringOne